### ISSUE TITLE
train UCF101 from scratch accuracy

### ISSUE THREAD

**Issue by JoeyHang** [USTC-Video-Understanding/I3D_Finetune#39]
Hi, Thanks a lot for your excellent work.
I have been trying to train UCF101 from scratch without Kinetics or ImageNet pretrained these days using your code. But something's wrong with the test accuracy.
When I trained about 33 epoches, the loss is around  0.1, train accuracy is around 93.6%, test accuracy is around 58%. But when i run 'test.py', the accuracy is only about 8% which is very weird.
I think the test data should be the same in your code 'finetune.py' and 'test.py', but accuracy is quite different. Or is there anything i've been always missing.
My only change to your code is changing 'saver.restore(sess, _CHECKPOINT_PATHS[train_data.mode+'_imagenet'])' line 211 in 'finetune.py' to '#saver.restore(sess, _CHECKPOINT_PATHS[train_data.mode+'_imagenet'])' for learning from scratch.
Please share your thoughts with me, or anyone faced same problem, please let me know.

**Comment by lhyfst** [comment]
@JoeyHang  Have you solved this problem?
